static/templates/popups/casting-item-create-dialog.hbs

```diff
@@ -0,0 +1,26 @@
+<form autocomplete="off" onsubmit="event.preventDefault();">
+    <label>{{ localize "PF2E.CastingItemCreateDialog.label" }}</label>
+    {{#if (ne itemType "cantripDeck5")}}
+    <div class="form-group">
+        <p><label>{{ localize "PF2E.CastingItemCreateDialog.spellLevelLabel" }}</label>
+        <select name="level" data-dtype="Number">
+        {{#select level}}
+            {{#each validLevels}}
+            <option value="{{this}}">{{this}}</option>
```

Comment:
you should probably match what heightenLevels does in spell-overlay.hbs

Reply:
Which part are you referring to? Using a named local instead of `this`? Or a localization string?

Reply:
I mean ordinal values like 3rd Level, 4th Level. But I'm looking at the current dialog again, and this isn't a regression, so its ok to leave as is.